dash/pages/profile.py

```diff
@@ -0,0 +1,60 @@
+import logging
+
+import dash_bootstrap_components as dbc
+import pandas as pd
+from util import get_authz, get_submission_service
+
+import dash
+from dash import Input, Output, callback, dash_table, html
+
+logger = logging.getLogger('dash')
+
+
+dash.register_page(__name__)
+
+
+def layout():
+    # query_service = get_submission_service()
+    # data = query_service.query("{ program(first:0) { id name projects { id code } } }")
+    # logger.error(data)
+
+    simplified_authz = {k: ','.join([m['method'] for m in v]) for k, v in get_authz().items()}
```

Comment:
kindof interesting that authorization information is put into a dataframe. Does this hold more than just auth information